packages/ripple-nuxt-tide/lib/layouts/default.vue

```diff
@@ -124,6 +124,13 @@ export default {
       this.anchorScrollFix(this.$route.hash)
     }
     RplLinkEventBus.$on('navigate', this.onNavigate)
+
+    if (window.MSInputMethodContext && document.documentMode) {
```

Comment:
@dylankelly I put it here since `window` property does not exist in nuxt.config.js as it server renders. Otherwise, happy to move this to config so this script loads regardless of browser type.

Reply:
@rubyllamadora You can check which environment you are in by using `process.client` or `process.server` https://nuxtjs.org/faq/window-document-undefined . So in this case a ternary check for process.client should work. Can you please move this out of the unrelated PR for border radius? I'd like to merge them separately, and its always a good practice to keep PR's only about one thing.

Reply:
I have reverted the change. I'll create new PR for this 👍